source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,251 @@
+
+Now we need to find an event that fits our case. Looking through the *event list* documentation, ``on player clicks 
+block`` looks like our best bet. We just have to adjust it for our specific use case by adding *optional* arguments. 
+According to the event syntax, it accepts a click type, a material and an area, which will just be ``left``, 
+``pumpkin`` and ``in Hub`` respectively.
+
+.. note::
+    This tutorial assumes our *Hub world* is in fact called ``Hub``.
+
+We'll just make sure it works for now, so we add it under the events key and end the line with a ``:``. Inside this 
+event we can place our first command. A good option for testing purposes is just to show a word in *chat*. The command 
+for this is :guilabel:`narrate`. As the documentation explains, its syntax is very simple, only requiring a message 
+argument. We feel the hype, so we'll go with a ``- narrate "yay"``.
+
+Our script should be the following at this point:
+
+.. code-block:: dscript
+    :name: figure_1_2
+    :linenos:
+    :emphasize-lines: 5,6
+
+    Halloween_Treasure_Hunt:
+    type: world
+    debug: true
+    events:
+      on player left clicks pumpkin in Hub:
+      - narrate "yay"
+
+.. rst-class:: figurecaption
+
+**Figure 1.2** Our world script with a specific event
+
+It's time to *save* the script file, *reload* scripts ingame with ``/denizen reload scripts`` and *trigger the event* 
+by left clicking a pumpkin block in our Hub world. We should now be able to see a cute little ``yay`` in chat, along 
+with some debug information in the console, just as we expected. That's great, but we also have to *test* and make 
+sure the event is not being triggered when clicking other types of blocks, when right clicking, or when clicking in 
+another world.
+
+Core Functionality
+==================
+
+We're ready to move further ahead and actually give a *reward* to the player clicking the block. Since we're nice 
+server owners, the prize will be a free *diamond*. This is where the :guilabel:`give` command comes in handy. Its 
+*syntax* specifies a single required argument: ``[money/xp/<item>|...]``. In our case, what we want to give the player 
+is a diamond item, so we can ignore the *money* and *xp* options.
+
+.. note::
+    When reading command documentation, It's important to keep in mind that anything inside ``< >`` is *not literal* 
+    and needs to be replaced. Arguments enclosed in ``[ ]`` are *required*, while ``( )`` means an argument is 
+    *optional*. The ``/`` symbol separates argument *options*, meaning you have to choose one of them.
+
+Let's go ahead and specify ``diamond`` as the first argument of our give command. We don't have to worry about who to 
+give the diamond to, as the command will target the linked player by default. That is, the player that triggered the 
+event. The full command line will then be ``- give diamond``.
```

Comment:
It's probably worth putting another code block figure here, showing the give command in context. It might not be totally useful/necessary, but it'll help the users tell where they should be.